hack/local-check-gh-automation.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+function cleanup() {
+  return_code=$?
+  os::cleanup::all
+  os::util::describe_return_code "${return_code}"
+  exit "${return_code}"
+}
+trap "cleanup" EXIT
+
+data="${BASETMPDIR}/data"
+mkdir -p "${data}"
+
+function OC() {
+	oc --context app.ci --namespace ci --as system:admin "$@"
```

Comment:
`--as system:admin`: I do not see the oc-cmd that we need it.

Reply:
I assumed that we needed that we needed admin perms to grab those secrets, but it looks like that is unnecessary. I'll correct that.